### DISCUSSION TITLE
post-name.html or index.html

### DISCUSSION THREAD

**Discussion by pprevos** [emacs-love/weblorg#67] [Q&A] [answered]
Hi,

I am looking for an alternative to Hugo and like the concept of weblorg.

Can it be configured to create a folder for each page and create an `index.html`?

So:

`this-is-a-post/`

instead of:

`this-is-a-post.html`

Regards
P:)

**Comment by guilhermecomum** [comment, marked as answer]
Hi @pprevos , thanks for your interest in weblorg, sorry the delay! 
It's easy to do this with weblorg :D
In the `publish.el` when define your post routes you can use
```
(weblorg-route
 :name "post"
 :input-pattern "posts/*.org"
 :template "post.html"
 :output "{{ slug }}/index.html"
 :url "/{{ slug }}")
```
And to link to the post page
`<a href="{{ url_for("post", slug=post.slug) }}">{{ post.title }}</a>`

**Reply by pprevos** [reply]
Thanks.

**Comment by pprevos** [comment]
Hi, I am working my way through moving my existing Hugo theme to Weblorg.

How do I copy my folder with images to the output folder? All examples I see hold the images in the themes folder. Can I copy static files from outside the themes folder?

The `weblorg-copy-static` seems to only source from the `themes` folder, which is OK for theme-related files (CSS), but not so much site-specific files such as images.

I tried using a route, but that only seems to work with org files. Can I create something that renders `[[url_for_img:images,file=sundown.png][orange]]`?

**Reply by guilhermecomum** [reply]
Hi @pprevos, 
Feel free to use my [blog](https://github.com/guilhermecomum/blog) as a reference any time.
To copy all static files you will need create a route for example:
``` elisp
;; route for static assets that also copies files to output directory
(weblorg-route
 :name "static"
 :url "/static/{{ file }}")
 ```
 and use to link the files
 ```elisp
 url_for_img:static,file=images/better-meetings/meeting-email.png
 ```
